wrangler-core/src/main/java/io/cdap/directives/column/Copy.java

```diff
@@ -74,33 +74,34 @@ public void destroy() {
 
   @Override
   public List<Row> execute(List<Row> rows, ExecutorContext context) throws DirectiveExecutionException {
-    for (Row row : rows) {
-      int sidx = row.find(source.value());
-      if (sidx == -1) {
-        throw new DirectiveExecutionException(NAME, String.format("Column '%s' does not exist.", source.value()));
-      }
-
-      int didx = row.find(destination.value());
-      // If source and destination are same, then it's a nop.
-      if (didx == sidx) {
-        continue;
-      }
-
-      if (didx == -1) {
-        // if destination column doesn't exist then add it.
-        row.add(destination.value(), row.getValue(sidx));
-      } else {
-        // if destination column exists, and force is set to false, then throw exception, else
-        // overwrite it.
-        if (!force) {
-          throw new DirectiveExecutionException(
-            NAME, String.format("Destination column '%s' already exists in the row. Use 'force' " +
-                                  "option to overwrite the column.", destination.value()));
-        }
-        row.setValue(didx, row.getValue(sidx));
-      }
-    }
-    return rows;
+    throw new DirectiveExecutionException("error!!!");
+//    for (Row row : rows) {
```

Comment:
Don't leave code commented out please.

Reply:
Whoops, I didn't mean to commit that! Fixed now.